tools/swift-inspect/Sources/swift-inspect/Operations/DumpRawMetadata.swift

```diff
@@ -42,5 +78,27 @@ internal struct DumpRawMetadata: ParsableCommand {
         }
       }
     }
+    
+    if metadataOptions.json {
+      let jsonStruct: RawMetadataOutput
+      let allocatorTagArray = Array(allocatorTagTotals.values).sorted(by: {$0.totalBytes > $1.totalBytes})
```

Comment:
If you change the dictionary as suggested above, then you can grab the keys and values together and sort them like:

```
let tagSizes = allocatorTagTotals.sorted(by: { $0.1 > $1.1 })
```

Then you can iterate like:

```
for (tag, size) in tagSizes {
```